index.html

```diff
@@ -278,20 +218,41 @@ <h4>Skills</h3>
                 </div>
                 <input type="text" name="full name" maxlength="30" placeholder="Full name" class="name" required>
                 <input type="email" name="email" placeholder="Email address" class="email" required>
-                <textarea name="comments" cols="30" rows="5" maxlength="500"
+                <textarea id="comments" name="comments" cols="30" rows="5" maxlength="500"
                     placeholder="Hey I’d like to connect to..."></textarea>
-                <input type="submit" value="Get in touch" class="get-button1">
+
+                <div class="flex">
+                    <input type="submit" value="Get in touch" class="get-button1">
+                    <p id="error"></p>
```

Comment:
- [ ] Please 🙏🏻 add a meaningful message and be more specific about what kind of error the user is getting. As I can see that it just prints "error" as the message which even doesn't tell what kind of error is and the user who will submit his/her query if they don't use the correct email then an error "Please add valid email"  will appear with some attractive colour like `#ff0000` color which can be easily visible to user and make the correction according to specific error.

### Your Output Looks Like

![image](https://github.com/Wils-FOURR-speed/My-portfolio/assets/66476329/4c8abf03-a627-40c1-826f-9766cb8e10c1)